lib/Alchemy/Phrasea/Controller/Root/LoginController.php

```diff
@@ -688,6 +688,10 @@ public function authenticationCallback(Request $request, $providerId)
         try {
             $provider->onCallback($request);
             $token = $provider->getToken();
+            $options = $provider->getOptions();
+            if (isset($options['base-url'])) {
+                $this->getSession()->set('auth_provider.base-url', $options['base-url']);
+            }
```

Comment:
you should not need to set this in session. you can fetch the provider with session('auth_provider.id'), then get options

Reply:
ok